app/controllers/admin/poll/questions/answers_controller.rb

```diff
@@ -49,11 +51,20 @@ def order_answers
     def answer_params
       documents_attributes = [:id, :title, :attachment, :cached_attachment, :user_id, :_destroy]
       attributes = [:title, :description, :question_id, documents_attributes: documents_attributes]
-      params.require(:poll_question_answer).permit(*attributes)
+      params.require(:poll_question_answer).permit(*attributes, *translation_params(params[:poll_question_answer]))
     end
 
     def load_answer
       @answer = ::Poll::Question::Answer.find(params[:id] || params[:answer_id])
     end
 
+    def resource
+      load_answer unless @answer
+      @answer
```

Comment:
Is this line necessary? As far as I know, Ruby always returns the value of the last statement but if you can provide some insight @microweb10, that'd be great :smile_cat:

Reply:
Sorry, @microweb10 may have copied that pattern from my banners PR.

The reason I explicitly set the return value is to avoid depending on the return value of `load_answer`, whose main purpose is to change state. I think it would be easy for someone to add more state-changing code and inadvertently change the return value, e.g.:

```ruby
def load_answer
  @answer = ::Poll::Question::Answer.find(params[:id] || params[:answer_id])
  Rails.logger.debug "Loaded answer #{@id.answer}"
end
```

Reply:
That is correct, @mlovic. Sorry about the inconvenience. As always, thanks for the detailed explanation! :bowing_man: